pywwt/core.py

```diff
@@ -197,10 +198,16 @@ def create_circle(self):
                        dec=self.get_center().dec.deg)
         return circle
 
-    def add_polygon(self):
+    def add_polygon(self,points=None):
```

Comment:
Here maybe you can also add ``**kwargs`` and pass that to ``Polygon`` below so that the annotation traits can also be set on initialization?